src/pages/index.js

```diff
@@ -134,45 +134,6 @@ function Home() {
             </div>
           </section>
         )}
-        <div className={clsx("hero hero--secondary", styles.heroBanner)}>
```

Comment:
Removing this entire section, because only 1 of 3 links in here applies to this version.